### PR TITLE
register cancel event for touch events

### PR DESCRIPTION
**Pull request by brianpeiris**
Fixes https://github.com/mozilla/hubs/issues/289

### CODE REVIEW

**Comment by johnshaughnessy** [comment]
This fix is surprising to me. I didn't realize that you could add more than one event name to bind to as they would be split by `", "` here: https://github.com/yoannmoinet/nipplejs/blob/2a1914e5e5b73e8c3d497a63e7017a6c12c507bc/src/utils.js#L37

That's why I added the `cancel` event to the pointer here: https://github.com/MozillaReality/nipplejs/pull/1/files#diff-c55f836adf5da3ded30afcfb1adeb357R23
But notice that the handler for cancel acts as if it was a pointend event because it calls `onAny` with `end`: https://github.com/MozillaReality/nipplejs/pull/1/files#diff-9e023b4b5b28995504b8e79efd9e3989R171

I wonder if my earlier fix should have just been, `end: 'pointerup, pointercancel'` and nothing else. I'm confused by why this PR fixes the issue.

**Comment by brianpeiris** [comment]
Yeah, I'm not sure if `end: 'pointerup, pointercancel'` would have been a better fix. But you'd still probably have to account for the `touchcancel` event. 
In any case, if you want to keep the `cancel` even, you need to register cancel events for `touch` as well as `pointer`, as this PR does, since the library will error when looking to bind `touchcancel` as detailed in mozilla/hubs#289.

File: src/start.js [hunk 1]
```diff
@@ -8,7 +8,8 @@ var events = {
     touch: {
         start: 'touchstart',
         move: 'touchmove',
-        end: 'touchend, touchcancel'
+        end: 'touchend, touchcancel',
```
Comment (johnshaughnessy) [inline src/start.js:11]: touchcancel can probably be removed here if included below

**Review by johnshaughnessy — APPROVED** [review]
Looks good - left 1 comment

**Pull request merged**